packages/reports/addon/templates/components/report-view.hbs

```diff
@@ -1,45 +1,64 @@
 {{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<NaviColumnConfig
+  @isOpen={{this.isColumnDrawerOpen}}
+  @drawerDidChange={{this.resizeVisualization}}
+/>
 <div class="report-view__visualization-main">
-  {{#if hasNoData}}
-    <div class="report-view__visualization-no-results">
-      No results available.
-    </div>
-  {{else}}
-    <div class="report-view__visualization-header">
-      <VisualizationToggle
-          @report={{@report}}
-          @validVisualizations={{this.validVisualizations}}
-          @onVisualizationTypeUpdate={{this.onVisualizationTypeUpdate}}
-      />
-        {{#animated-if (and this.hasRequestRun (not this.isEditingVisualization)) use=this.fadeTransition}}
-          <div class="clickable report-view__visualization-edit-btn" role="button" {{on "click" this.toggleEditVisualization}}>
-            Edit {{this.visualizationTypeLabel}}
-            <NaviIcon @icon="pencil" />
-          </div>
-        {{/animated-if}}
-        {{#animated-if (not this.hasRequestRun) use=this.fadeTransition}}
-          <div class="report-view__info-text">
-          <NaviIcon @icon="exclamation-circle" /> Run request to update {{this.visualizationTypeLabel}}</div>
-        {{/animated-if}}
-    </div>
-    <ReportVisualization
-      class="report-view__visualization"
-      @report={{@report}}
-      @response={{@response}}
-      @container={{this}}
-      @annotationData={{this.annotationData}}
-      @isEditing={{and this.isEditingVisualization @hasRequestRun}}
-      @onUpdateReport={{pipe
-        (route-action "onUpdateReport")
-        (route-action "validate" @report)
-        (route-action "runReport" @report)
-      }}
-    />
-    <MissingIntervalsWarning
-      @response={{this.response}}
-      @onDetailsToggle={{this.resizeVisualization}}
+  <div class="report-view__visualization-header">
```

Comment:
div -> section

Reply:
Not sure `<section>` is the right tag here. 

> section denotes a semantic section of your content to help construct a document outline